packages/SwingSet/src/lib-nodejs/engine-gc.js

```diff
@@ -7,8 +7,11 @@ if (typeof bestGC !== 'function') {
   // Node.js v8 wizardry.
   v8.setFlagsFromString('--expose_gc');
   bestGC = vm.runInNewContext('gc');
-  // Hide the gc global from new contexts/workers.
-  v8.setFlagsFromString('--no-expose_gc');
+  // we leave --expose_gc turned on, otherwise AVA's shared workers
+  // may race and disable it before we manage to extract the binding
```

Comment:
Should we leave a comment that it's safe since at worse it would just leave a global `gc` in new realms (which we don't create), and that would get removed in Compartments anyway.